pisa/core/distribution_maker.py

```diff
@@ -275,6 +292,43 @@ def source_code_hash(self):
     def hash(self):
         return hash_obj([self.source_code_hash] + [p.hash for p in self])
 
+    @property
+    def num_mc_events_per_bin(self):
+        '''
+        returns an array of bin indices where none of the 
+        pipelines have MC events
+
+        assumes that all pipelines have the same binning output specs
+        '''
+        num_bins = self.pipelines[0].stages[-1].output_specs.tot_num_bins
+        self.num_mc_events_per_bin = np.zeros(num_bins)
+
+        for p in self.pipelines:
+
+            for c in p.stages[-1].data:
+
+                # skip data or pseudo-data pipelines
+                if c.name == 'data' or c.name == 'Total':
```

Comment:
is this a good idea to treat data, or what you think is pesudo data because the name is "Total" any different?
I'd rather not make this distinction here. And for (pseudo)data you're anyway only interested in expected values.....so having extra info won't hurt.

What if i have a pipeline of my model called "Total".....your code will then unexpectedly not work. Or what if my pseudodata is called "total" and not "Total", then behaviour will be different

Reply:
Hi Etienne, test test

Reply:
testing on my end!

Reply:
wakawaka